service/src/main/java/uk/nhs/adaptors/gpc/consumer/filters/SslContextBuilderWrapper.java

```diff
@@ -54,6 +54,12 @@ public SslContext buildSSLContext() {
         return io.netty.handler.ssl.SslContextBuilder.forClient().build();
     }
 
+    @SneakyThrows
+    public SslContext buildSSLContextForSds() {
+        LOGGER.info("Using standard SSL context.");
+        return io.netty.handler.ssl.SslContextBuilder.forClient().build();
+    }
+
```

Comment:
Instead of "leaking" information about SDS into this interface it could be made more generic / reusable.

```suggestion
        return buildStandardSslContext()
    }

    @SneakyThrows
    public SslContext buildStandardSslContext() {
        LOGGER.info("Using standard SSL context.");
        return io.netty.handler.ssl.SslContextBuilder.forClient().build();
    }

```